libraries/ui/src/Modal.tsx

```diff
@@ -25,7 +25,7 @@ export const Modal: React.FC<ModalProps> = ({
       isDismissable
       isOpen={isOpen}
       onOpenChange={setIsOpen}
-      className="fixed inset-0 z-10 overflow-y-auto bg-black/25 flex min-h-full items-center justify-center p-4 backdrop-blur-xs"
+      className="fixed inset-0 z-60 overflow-y-auto bg-black/25 flex min-h-full items-center justify-center p-4 backdrop-blur-xs"
```

Comment:
Previously the header would appear on top of this on small screens. I assume this wasn't intentional